cmd/monaco/deploy/deploy.go

```diff
@@ -287,6 +287,10 @@ func validateAuthenticationWithProjectConfigs(projects []project.Project, enviro
 							return fmt.Errorf("API of type '%s' requires a token for environment '%s'", conf.Type, envName)
 						}
 					case config.SettingsType:
+						t, ok := conf.Type.(config.SettingsType)
```

Comment:
Technically the `ok` is not needed becasue the switch-case already checks it, but also fine if we leave this one